CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ Compatibility:
 * Support refinements for Kernel#respond_to? (#2120, @ssnickolay)
 * JCodings has been updated from 1.0.45 to 1.0.55.
 * Joni has been updated from 2.1.30 to 2.1.40.
+* Implement `String#undump` (#2131, @kustosz)
```

Comment:
This will need to in the next release 21.0. I'll have a PR soon adding that to the CHANGELOG, so let's rebase this one after.

Reply:
That PR is merged, if you rebase you should see the new `# 21.0.0` in the CHANGELOG.